src/views/CoCurricularTranscript/guestTranscript.tsx

```diff
@@ -0,0 +1,116 @@
+import { MembershipHistory, MembershipHistorySession } from 'services/user';
+import { StudentProfileInfo } from 'services/user';
+import { Class } from 'services/peopleSearch';
+import { Participation } from 'services/membership';
+import { TranscriptItems } from 'services/transcript';
+
+const participation1: Participation = Participation.Member;
+const currentYear: string = new Date().getFullYear().toString();
+const session1: MembershipHistorySession = {
+  MembershipID: 2904,
+  SessionCode: currentYear + '01',
+  Participation: participation1,
+};
+const activity1: MembershipHistory = {
+  ActivityCode: 'ISO',
+  ActivityDescription: 'International Student Organization',
+  ActivityImagePath: '',
+  Sessions: [session1],
+  LatestDate: currentYear + '-05-10T00:00:00',
+};
+const participation2: Participation = Participation.Member;
+const session2: MembershipHistorySession = {
+  MembershipID: 6547,
+  SessionCode: currentYear + '09',
+  Participation: participation2,
+};
+const activity2: MembershipHistory = {
+  ActivityCode: '360',
+  ActivityDescription: '360.gordon.edu',
+  ActivityImagePath: '',
+  Sessions: [session2],
+  LatestDate: currentYear + '-12-15T00:00:00',
+};
+export const transcriptItems1: TranscriptItems = {
+  honors: [],
+  experiences: [],
+  service: [],
+  activities: [activity1, activity2],
+};
+export const student1: StudentProfileInfo = {
+  fullName: 'First M Last',
+  Majors: ['Undesided'],
+  Minors: [],
+  Advisors: [],
+  ID: '50232456',
+  Title: '',
+  FirstName: 'First',
+  MiddleName: '',
+  LastName: 'Last',
+  Suffix: '',
+  MaidenName: '',
+  NickName: '',
+  OnCampusBuilding: 'WIL',
+  OnCampusRoom: '320',
+  OnCampusPhone: '',
+  OnCampusPrivatePhone: '',
+  OnCampusFax: '',
+  Mail_Location: '778',
+  HomeStreet1: '255 Grapevine Rd',
+  HomeStreet2: '',
+  HomeCity: 'Wenham',
+  HomeState: 'MA',
+  HomePostalCode: '',
+  HomeCountry: 'AM',
+  HomePhone: '',
+  HomeFax: '',
+  KeepPrivate: '',
+  Barcode: '72639526863451',
+  Email: 'first.last@gordon.edu',
+  Gender: 'F',
+  AD_Username: 'first.last',
+  show_pic: 0,
+  preferred_photo: 0,
+  Country: 'America',
```

Comment:
I don't think this is a valid country. I believe it would be either `US` or `United States of America`.

Reply:
Okay! Changing it now